crates/proof-of-sql/tests/extra_coverage_tests.rs

```diff
@@ -0,0 +1,111 @@
+#![cfg(test)]
+//! This file is added to improve test coverage of various modules in proof-of-sql.
```

Comment:
In general please put your tests in places other than a file named `extra_coverage_tests.rs`.

Reply:
See https://github.com/spaceandtimelabs/sxt-proof-of-sql/blob/main/crates/proof-of-sql/src/base/database/column.rs#L657-L1204
and
https://github.com/spaceandtimelabs/sxt-proof-of-sql/blob/main/crates/proof-of-sql/src/base/database/expression_evaluation_test.rs
for examples of where our tests are located.